src/authentication/background/auth-background.ts

```diff
@@ -0,0 +1,49 @@
+import { AuthService } from 'src/authentication/background/auth-service'
+import {
+    FirebaseFunctionsAuth,
+    FirebaseFunctionsSubscription,
+} from 'src/authentication/background/firebase-functions-subscription'
+import { remoteEventEmitter } from 'src/util/webextensionRPC'
+import { AuthFirebase } from 'src/authentication/background/auth-firebase'
+import { MockAuthImplementation } from 'src/authentication/background/mocks/auth-mocks'
+
+export class AuthBackground {
+    authService: AuthService
+    subscriptionServerFunctions: FirebaseFunctionsSubscription
+    authServerFunctions: FirebaseFunctionsAuth
+
+    constructor(
```

Comment:
Generally I try to pass in dependencies in a single object, just because of lower overhead in terms of the ordering of positional arguments.